cpp/src/arrow/ipc/writer.h

```diff
@@ -313,6 +314,16 @@ ARROW_EXPORT
 Status GetDictionaryPayloads(const Schema& schema,
                              std::vector<std::unique_ptr<IpcPayload>>* out);
 
+/// \brief Compute IpcPayload for the given schema
+/// \param[in] schema the Schema that is being serialized
+/// \param[in,out] pool for any required temporary memory allocations
+/// \param[in,out] dictionary_memo TODO
```

Comment:
"TODO"? Does it mean the docstring needs to be completed or that the DictionaryMemo argument is ignored?